Dockerfile

```
@@ -1,10 +1,13 @@
-FROM python:3.11
+FROM python:3.12-slim
+# Ensure poetry does not create a virtualenv
+ENV POETRY_VIRTUALENVS_CREATE=false
 
```

When using the "slim" version of the python base image we don't have curl out of the box:

```suggestion
RUN apt-get update -yqq \
  && apt-get install -yqq --no-install-recommends curl \
  && apt-get -q clean \
  && rm -rf /var/lib/apt/lists
```

---

Good catch! I'll just switch to the regular 3.12 container.